src/annotator/highlight-clusters.tsx

```diff
@@ -125,11 +133,18 @@ export class HighlightClusterController implements Destroyable {
     this._render();
   }
 
+  /**
+   * Set a value for an individual CSS variable at :root
+   */
+  _setClusterStyle(key: string, value: string) {
+    document.documentElement.style.setProperty(key, value);
```

Comment:
This is outside the scope of this PR, but I wonder if it would make sense to set cluster styles on the root element passed to this controller rather than the document element. Are there any special consequences of applying styles to the root element which would be lost if applying them to a descendant instead?

Reply:
I guess one reason is that the static CSS applies defaults on `:root`, and applying the dynamic updates on `:root` as well means that you can see the values from the static CSS plus the same overrides when looking at the same HTML element in devtools.

Reply:
I'm not sure how one would orchestrate setting CSS values on an element determined at runtime with the initial values of several related variables that are set in `highlights.scss` on `:root`. It may be a non-issue; I'll look into it.